### PR TITLE
Fix UserController Tests to pass. Fix Inventory Repositories

### LINKED ISSUE #4: Set Up Project
create repo
connect to intelliJ
connect to azure
set up CI

### PR DESCRIPTION
**Pull request by thanujasiva**
Leaving the pom.xml as is, so it still runs the unit tests.

NOTE: the SpringBoot tests (ex UserControllerTest) are not properly run in the maven packaging/CI.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,12 +78,6 @@
           <artifactId>org.eclipse.persistence.jpa</artifactId>
           <version>2.5.0</version>
       </dependency>
-      <dependency>
-          <groupId>org.xerial</groupId>
```
Comment (thanujasiva) [inline pom.xml:94]: This dependency duplicated the first dependency

**Comment by thanujasiva** [comment]
Not ready to merge yet. The CI is failing, not sure why (packaging worked fine locally).

I think there are known issues with the main branch (due to PR https://github.com/waheebh1/online-bookstore/pull/13). This branch does not include those changes though.

File: src/main/java/bookstore/inventory/BookRepository.java [hunk 2]
```diff
@@ -10,8 +10,8 @@
 import java.util.List;
 
 public interface BookRepository extends CrudRepository<Book, String>{
-    Book findBIsbn(String isbn);
+    Book findByIsbn(String isbn);
     List<Book> findByTitle(String title);
     List<Book> findByGenre(String genre);
-    List<Book> findByAuthor(ArrayList<Author> author);
+    //List<Book> findByAuthor(ArrayList<Author> author); // FIXME - causes issues with starting SpringBoot
```
Comment (thanujasiva) [inline src/main/java/bookstore/inventory/BookRepository.java]: This was the main issue causing SpringBoot (App.java, UserControllerTest.java) to not run properly

**Review by maishabd23 — APPROVED** [review]

File: src/main/java/bookstore/inventory/BookRepository.java (hunk 2 above)
Comment (thanujasiva) [reply src/main/java/bookstore/inventory/BookRepository.java]: Fixed by changing to `Collection`

**Pull request merged**